remotes/docker/fetcher.go

```diff
@@ -309,5 +314,32 @@ func (r dockerFetcher) open(ctx context.Context, req *request, mediatype string,
 		}
 	}
 
-	return resp.Body, nil
+	body := resp.Body
+	encoding := strings.FieldsFunc(resp.Header.Get("Content-Encoding"), func(r rune) bool {
+		return unicode.IsSpace(r) || r == ','
```

Comment:
```suggestion
		return r == '' " || r == ' \t' || r == ','
```

To avoid matching U+3000, etc.

Reply:
@ndeloof Can you take a look?

Reply:
not actively working on this topic, feel free to adopt/revisit/reimplement this PR